sdk/containerregistry/arm-containerregistry/src/containerRegistryManagementClientContext.ts

```diff
@@ -13,7 +13,7 @@ import * as msRest from "@azure/ms-rest-js";
 import * as msRestAzure from "@azure/ms-rest-azure-js";
 
 const packageName = "@azure/arm-containerregistry";
-const packageVersion = "7.0.0";
+const packageVersion = "8.0.0";
```

Comment:
because the api version is "package-2019-12-preview", the SDK should be preview too, isn't it?

Reply:
Currently there's no preview tag in JS SDK

Reply:
There is nothing stopping us from using previews in the package version. The main question would be as to what tag would the release get. In the data plane libraries, when we want to release preview packages for a package that is already gone GA, we add the tag `next` to the release while retaining the tag `latest` for the GA release. This way, users by default will only get the stable version. People wanting to get the preview version should explicitly use that version.

Reply:
@ramya-rao-a Thanks for the information, @RodgeFu Should we also take data-plane strategy now? I think the current mgmt plane release pipeline doesn't support to release with tags?

Reply:
yes, i think we can do the same as data-plane. Let's add that support then. :)

Reply:
One thing to keep in mind is what the expectation from the service team is. If their expectation is that this preview version should be the default version everybody should get, then we cannot take the data plane strategy to add the `next` tag

Reply:
Yes. That's because we can't support multi-api like python SDK at this moment, and we have to choose a tag in their readme.md to release. so the preview tag is just like offical  version.

Reply:
Ok, in that case, let's go with non preview version